sdk/go/auto/stack.go

```diff
@@ -1573,7 +1573,7 @@ func (s *languageRuntimeServer) Run(ctx context.Context, req *pulumirpc.RunReque
 				if pErr, ok := r.(error); ok {
 					err = fmt.Errorf("go inline source runtime error, an unhandled error occurred: %w", pErr)
 				} else {
-					err = errors.New("go inline source runtime error, an unhandled error occurred: unknown error")
+					err = errors.New("go inline source runtime error, an unhandled error occurred: unknown error: %v", r)
```

Comment:
```suggestion
					err = fmt.Errorf("go inline source runtime error, an unhandled error occurred: unknown error: %v", r)
```

I think this change is needed for this to compile and print the error correctly.

Reply:
```suggestion
					err = fmt.Errorf("go inline source runtime error, an unhandled panic occurred: %v", r)
```
Instead maybe?

Reply:
woops, thanks